lib/plugin.js

```diff
@@ -71,133 +77,124 @@ module.exports = {
               prevent = true
               node.remove()
           }
+
           return prevent
         },
-        end: function (node, metadata, context) {
+        end (node, metadata, context) {
           return true
         }
       },
       options: {
         expect: { self: true },
         stack: [],
-        begin: function (node, metadata, context) {
+        begin (node, metadata, context) {
           this.stack.push(util.extend({}, context.config))
           let options
           try {
             options = JSON.parse(metadata.param)
           } catch (e) {
-            throw node.error('Invlaid options object', { details: e })
+            throw node.error('Invalid options object', { details: e })
           }
+
           context.config = config.configure(options, context.config.plugins)
           context.util = util.configure(context.config)
           return true
         },
-        end: function (node, metadata, context) {
+        end (node, metadata, context) {
           const config = this.stack.pop()
           if (config && !metadata.begin) {
             context.config = config
             context.util = util.configure(context.config)
           }
+
           return true
         }
       },
       config: {
         expect: { self: true },
         expr: {
-          fn: /function([^(]*)\(([^()]*?)\)[^{]*\{([^]*)\}/ig,
+          fn: /function([^(]*)\(([^()]*?)\)[^{]*{([^]*)}/ig,
           rx: /\/([^/]*)\/(.*)/ig
         },
```

Comment:
@XhmikosR I believe these are unused.

Reply:
Not sure, you know better for sure :) You could remove them in a separate patch later.